migrations/20240318035358_erc721_contract_init.ts

```diff
@@ -0,0 +1,18 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('erc721_contract', (table) => {
+    table.increments();
+    table.integer('evm_smart_contract_id').notNullable().unique();
+    table.string('address').unique().notNullable();
+    table.string('symbol').index();
+    table.string('name');
+    table.boolean('track').defaultTo(false).index();
+    table.integer('last_updated_height').index();
```

Comment:
total_supply?

Reply:
có khi thêm no_holders luôn, trước là ở đây hay tách nhỉ?

Reply:
Theo standard của ERC71 thì ko query được total_supply, total_supply và no_holders sẽ làm khi mình có bảng holders anh nhé, mình sẽ tự tính toán ạ